src/components/Form/types.ts

```diff
@@ -51,10 +51,16 @@ type InputWrapperProps<TInput extends ValidInputs> = Omit<BaseInputProps, 'ref'>
 type ExcludeDraft<T> = T extends `${string}Draft` ? never : T;
 type OnyxFormKeyWithoutDraft = ExcludeDraft<OnyxFormKey>;
 
+type DraftOnly<T> = T extends `${string}Draft` ? T : never;
+type OnyxFormKeyOnlyDraft = DraftOnly<OnyxFormKey>;
```

Comment:
```suggestion
type DraftOnly<T> = T extends `${string}Draft` ? T : never;
type OnyxFormKeyWithDraft = DraftOnly<OnyxFormKey>;
```

Reply:
This proposed name suggests that could be something more than Draft, but my intention this key should have keys with Draft suffix only